lib/shipcloud/version.rb

```diff
@@ -1,3 +1,3 @@
 module Shipcloud
-  VERSION = "0.11.0".freeze
+  VERSION = "0.12.0".freeze
```

Comment:
The adaptation of the version is done in an extra release step.

```suggestion
  VERSION = "0.11.0".freeze
```